lib/updateDeps.js

```diff
@@ -306,7 +328,7 @@ const updateManifestDeps = (pkg, writeOut = true, bumpStrategy = "override", pre
 		scopes.forEach((scope) => {
 			if (scope[dependency.name]) {
 				scope[dependency.name] = resolveNextVersion(
-					get(dependency, "_lastRelease.version", "0.0.0"),
+					scope[dependency.name],
```

Comment:
I think this is the bugfix of `inherit` 🤔

Reply:
Seems like that, so there's no need for yet another br change.